core/videocache_test.go

```diff
@@ -43,30 +43,7 @@ func TestEvictMasterPlaylist(t *testing.T) {
 	}
 }
 
-func TestGetHLSSubscriber(t *testing.T) {
```

Comment:
This tests GetHLSSubscriber and it got removed

Reply:
Good point.  Added back in.